### PR TITLE
Budgetly App

### PR DESCRIPTION
**Pull request by karelvanoordt**
App created for Ruby on Rails module Capstone Project:

[Live Loom Presentation](https://www.loom.com/share/23dc43c44bb445778e636a0c295f34e9?sharedAppSource=personal_library)

[Demo link](https://desolate-peak-95618.herokuapp.com/)

- Mobile First designed app
- App has the following features:
- - User can sign up and log in with created credentials.
- - User can create expense categories and specific expenses.
- - User can see listed categories and the total amount of each one.
- Created tests using Capybara and RSpec.

### CODE REVIEW

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑⛔**
Hello @karelvanoordt 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- When I click on the live demo link it gives me this error
ror
Please make sure you fix that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by karelvanoordt** [comment]
Hi @emmanuellekamwa, thanks for taking your time to review the project.

I have a concern, on my end the link seems to be working fine. Could you try again?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+- GitHub: [@karelvanoordt](https://github.com/karelvanoordt)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/karelvanoordt/)
+
+
+**Aknowledgment**
+
+- Big thanks to [Gregoire Vella](https://www.behance.net/gregoirevella) for providing the design guidelines for this project.
+
+
```
Comment (ahmetbozaci) [inline README.md]: ### OPTIONAL
Nice to add a design link in the acknowledgment section. 
You can fix the typo `Aknowledgment`
- [ ] Please also create a license section and add [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)

[Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 


@ ✋🏼 
### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

aproved

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**